src/components/specific/files/document-viewer/DocumentViewer.vue

```diff
@@ -76,6 +78,18 @@ const file = computed(() => {
     case ".svg":
     case ".webp":
       return doc.file;
+    case ".ppt":
```

Comment:
Maybe the switch is not relevant here as you have to repeat all `OFFICE_FILES`. Use `arr.some` instead ?